docs/source/concept_models/vivarium_swissre_lungcancer/concept_model.rst

```diff
@@ -220,7 +220,7 @@ Additional considerations:
 
 .. image:: concept_model_diagram.svg
 
-Note that we are not modeling an effect of lung cancer screening coverage on lung cancer mortality and morbidity because it is not an explicit outcome of interest in this project, although there is evidence that there is an effect.
+Note that we are not modeling an effect of lung cancer screening coverage on lung cancer mortality because it is not an explicit outcome of interest in this project, although there is evidence of an effect.
```

Comment:
Internally we can track the deaths due to TBL cancer and compare the impact of screening on mortality to external evidences.